### PR TITLE
Daemon: Updated log messages in create endpoint.

### PR DESCRIPTION
**Pull request by eloycoto**
To make easier to debug send the correct logger all log messages from
create-endpoint.

Signed-off-by: Eloy Coto <eloy.coto@gmail.com>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/7268)

### CODE REVIEW

**Comment by eloycoto** [comment]
test-me-please

File: daemon/endpoint.go [hunk 1]
```diff
@@ -233,32 +233,34 @@ func (d *Daemon) createEndpoint(ctx context.Context, epTemplate *models.Endpoint
 		}
 	}
 
+	logger := ep.Logger(daemonSubsys)
 	if err := endpointmanager.AddEndpoint(d, ep, "Create endpoint from API PUT"); err != nil {
-		log.WithError(err).Warn("Aborting endpoint join")
+		logger.WithError(err).Warn("Aborting endpoint join")
 		return nil, err
 	}
```
Comment (tgraf) [inline daemon/endpoint.go:242]: You want to call `ep.Logger()` again here so the log messages will have the endpoint ID populated. It is assigned in `endpointmanager.AddEndpoint()`

**Review by tgraf — CHANGES_REQUESTED** [review]

File: daemon/endpoint.go (hunk 1 above)
Comment (eloycoto) [reply daemon/endpoint.go:242]: good one, fixed ;-)

**Comment by eloycoto** [comment]
test-me-please

**Review by tgraf — APPROVED** [review]

**Review by nebril — APPROVED** [review]

**Pull request merged**